src/infra/docs/bors.md

```diff
@@ -51,23 +48,43 @@ There are multiple steps needed to add a repository to our Bors instance:
    Make sure to replace `[ALL, OTHER, JOBS]` with a list of all the jobs you
    want to gate on.
 
-3. Add the repository name to the `permissions!` macro in the [team
+   These jobs need to run on specific branches (`auto` and `try`) so it's necessary
+   to add those branches to the list of branches tested by the CI provider. For GitHub
+   Actions that looks like this:
+
+   ```yaml
+   on:
+      push:
+          branches: [ 
+            master, # This is any branch you normally run CI on
```

Comment:
Just curious, is there a particular reason to include `master` here?  Usually it shouldn't be necessary to run CI on master, since bors just ran the full test suite on the `auto` branch.  Unless you have special requirements (like `rust-lang/rust` which just does toolstate stuff on master), that essentially ends up running the same tests on the same commit twice.

Reply:
I think this is just an example, but `master` might be an unfortunate name (as it's one of the common default branches names, so people might expect the default branch has to be tested). Maybe replacing this with a commented `additional-branch`?